src/types/Ctype.ts

```diff
@@ -1,12 +1,46 @@
 import * as sdk from '@kiltprotocol/sdk-js'
 
-export interface ICType {
+export interface ICType extends sdk.ICTypeMetadata {
   cType: sdk.ICType
   metaData: {
     author: string
   }
 }
 
+export interface CTypeMetadata {
+  cType: {
+    schema: sdk.CType['schema']
+    metadata: sdk.CTypeMetadata['metadata']
+    owner: sdk.CType['owner']
+    hash: sdk.CType['hash']
+  }
+  metaData: {
+    author: string
+  }
+}
+
+export interface ICTypeInput {
+  $id: string
+  $schema: string
+  properties: object[] // TO DO: need to refine what properties are
+  required: string[]
+  title: string
+  description?: string
+  type: string
+  owner?: string
+}
+
+export interface IClaimInput {
+  $id: string
+  $schema: string
+  properties: object
+  required: string[]
+  title: string
+  description?: string
+  type: string
+  owner?: string
+}
```

Comment:
I had taken all the information from the sdk CTypeUtilites and added it here. 

Additionally, I created a new typeface to take the combined schema and metadata to match what we previously had. 

It was a workaround in order not to take up too much time.